common/Scripts/helpers.sh

```diff
@@ -1,30 +1,49 @@
 #!/usr/bin/env bash
 
+# shellcheck wants to add '|| exit 1' willy nilly, so let's
```

Comment:
This needs to be reworded. Any comment starting with `#shellcheck` is interpreted by shellcheck as a directive.